src/main/java/frc/robot/systems/DriveFSMSystem.java

```diff
@@ -703,7 +754,153 @@ private void handleWaitToReceiveBallsState(TeleopInput input, double waitTime) {
 	 * @param path the path to run
 	 */
 	public void setAutoPath(DesiredMode path) {
-		autoPath = path;
+		switch (path) {
+			case RED_3_BALL:
+				startAngle = Constants.PP_R3_HUB_ANGLE_DEG;
```

Comment:
Would be nice to have a wrapper class around these auto parameters. I.e. here we'd do currentAuto = <some object>, and whenever we need to reference a value, we can just do currentAuto.target_angle or something like that.